src/Compilers/CSharp/Test/Syntax/Parsing/ParserRegressionTests.cs

```diff
@@ -107,5 +108,35 @@ public void ThrowInInvocationCompletes()
 
             SyntaxFactory.ParseExpression(code);
         }
+
+        [Fact(Skip = "https://github.com/dotnet/roslyn/issues/13719")]
```

Comment:
This is the long-standing convention for unit tests: `Skip` points to a bug report, and a `WorkItem` attribute is added when it is fixed (it might be this issue or it might be another). I'd rather continue with the current conventions.

---
In reply to: [78419151](https://github.com/dotnet/roslyn/pull/13720#discussion_r78419151) [](ancestors = 78419151)